entity-framework/core/querying/complex-query-operators.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Complex Query Operators - EF Core
+author: smitpatel
+ms.date: 10/03/2019
+ms.assetid: 2e187a2a-4072-4198-9040-aaad68e424fd
+uid: core/querying/complex-query-operators
+---
+# Complex Query Operators
```

Comment:
We probably need a note that this page applies only to relational database.

Reply:
That is incorrect.

Reply:
Care to elaborate?

Reply:
The page is not necessarily put here for relational. We can add more operators here as we go. Like Cosmos can translate bunch of SelectMany in awesome way. The main purpose in not putting relational here is to reuse this page as needed for other providers in future. Further, all the operators talk about the relational translation so it is evident that it applies to relational only.

Reply:
Re Cosmos translations specifically, we may want to consider talking about that in the Cosmos provider docs - not sure it's a good idea to pack all the provider-specific info in one page. It will not be relevant for most readers of this page. Otherwise the page as it is seems 100% relational to me, but I don't have super strong feelings on this.

Reply:
Yeah, there is one point on this long and elaborate page that is also common to non-relational providers.

As I said if you feel very strongly about not including a note on this, I definitely won't die on that hill.